File: src/basics/patterns/Pattern1.java
package src.basics.patterns;

public class Pattern1 {
    /*
    Pattern 1:
    https://practice.geeksforgeeks.org/problems/square-pattern/1

     * * * *
     * * * *
     * * * *
     * * * *

     */
    static void printSquare(int n) {
        // code here
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= n; j++) {
                System.out.print("* ");
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        printSquare(5);
    }
}

File: basics/patterns/pattern1.py

# Pattern 1:
# https://practice.geeksforgeeks.org/problems/square-pattern/1
# 
# * * * * *
# * * * * *
# * * * * *
# * * * * *
# * * * * *

def printTriangle(N):
    for _ in range(N):
        for _ in range(N):
            print("*", end=" ")
        print("")


if __name__ == "__main__":
    printTriangle(5)
